src/main/java/com/redhat/cloud/notifications/routers/UserConfigService.java

```diff
@@ -195,9 +205,7 @@
         return Uni.createFrom().deferred(() -> {
             final SettingsValues values = new SettingsValues();
 
-            return bundleResources.getBundles()
-                    // Todo: Create a method that gets the bundle by name
-                    .filter(bundle -> bundle.getName().equals(bundleName))
+            return bundleResources.getBundle(bundleName)
                     .onItem().transformToMulti(bundle -> {
```

Comment:
If you apply the suggestion from my previous comment, this may return a `Uni` containing a `null` value.

Replacing `.onItem().transformToMulti(bundle -> {` with `.onItem().ifNotNull().transformToMulti(bundle -> {` should be enough to handle the `null` value but it needs to be confirmed.

Reply:
I'll check